<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypeInStage</class>
 <widget class="QWidget" name="TypeInStage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="speakButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Pronounce the word</string>
       </property>
       <property name="icon">
        <iconset resource="pixmaps/pixmaps.qrc">
         <normaloff>:/icons/speaker.png</normaloff>:/icons/speaker.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="wordEdit">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="correctLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="enterButton">
     <property name="text">
      <string>I entered the word (Enter)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextWordButton">
     <property name="text">
      <string>Next word (Enter)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextStageButton">
     <property name="text">
      <string>Next stage (Enter)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pixmaps/pixmaps.qrc"/>
 </resources>
 <connections/>
</ui>
